uiowa/uiowa
# Feature: Residence halls page with filters

Resolves https://github.com/uiowa/uiowa/issues/5350. 

# How to test

- `blt frontend`
- `ddev blt ds --site=housing.uiowa.edu`
- Go to https://housing.uiowa.ddev.site/residence-halls and test filters and verify that the page looks like https://github.com/uiowa/uiowa/issues/5350#issuecomment-1177923328. 
- Test on https://admissions.uiowa.ddev.site/areas-of-study and https://admissions.uiowa.ddev.site/kiosk check that filters still look decent. 

# Todo
- [ ] Get updated intro copy from @meganyoder
